style.css

```diff
@@ -0,0 +1,61 @@
+body{
+    padding: 0;
+    margin: 1.7em 0.6em;
+    box-sizing: border-box;
+}
+.body{
+    background-color: #EEE;
+    padding:0.5em 1em;
+    border-radius: 5px;
+
+}
+header{
+    text-align: center;
+}
+header h1{
+    width: 100%;
+    margin: 0.7em auto 0;
+    padding: 0.6em 0;
+    font-size: 2.2em;
+}
+header p{
+    margin: 0 auto;
+    padding: 1em 0 1.3em;
+}
+
+div #img-div {
+    text-align: center;
+    display: block;
+    background-color: #FFF;
+}
```

Comment:
- [ ] Great job however it is better to always use the class attribute for multiple elements instead of use ID selector